### PR TITLE
Ref task #2547 User bookmark movie part 2

### PR DESCRIPTION
**Pull request by ndviet2307**

### CODE REVIEW

**Comment by ndviet2307** [comment]
screenshot from 2018-10-16 10-59-27

**Comment by ndviet2307** [comment]
screenshot from 2018-10-16 11-04-30

**Comment by lyvanminh** [comment]
OK

**Comment by ndviet2307** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -195,3 +195,12 @@ html,body {
 .pagination{
   justify-content: center !important;
 }
+
+.bookmark-link{
```
Comment (baclv) [inline app/assets/stylesheets/custom.scss]: thêm dấu cách trước `{` bạn nhé

**Comment by ndviet2307** [comment]
screenshot from 2018-10-16 14-30-55

**Comment by ndviet2307** [comment]
ready

**Pull request merged**